CMakeLists.txt

```diff
@@ -45,17 +45,6 @@ add_c_compiler_flag(-O0 DEBUG)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 
-# Custom command produces build_meta_data.c, and a dummy output that's
-# not actually produced, in order to force Version.cmake to always be
-# re-run before the build
-ADD_CUSTOM_COMMAND(
-    OUTPUT  ${CMAKE_CURRENT_BINARY_DIR}/build_meta_data.c
-            ${CMAKE_CURRENT_BINARY_DIR}/_build_meta_data.c
-    COMMAND ${CMAKE_COMMAND} -P
```

Comment:
please try tdnf --version with your changes. also examine build_metadata.c after the make and you will see that the version string is populated empty. 
why would there be a version change after cmake is executed in a ci run?

Reply:
> please try tdnf --version with your changes

Here is my output with the version output. This is soon after a cmake.
```
Scanning dependencies of target tdnf
[ 97%] Building C object CMakeFiles/tdnf.dir/tools/cli/main.c.o
[100%] Linking C executable bin/tdnf
[100%] Built target tdnf
root@photon-ea4f47d2e8d0 [ ~/tdnf/build ]# cat build_meta_data.c
const char *PACKAGE_NAME = "tdnf";
const char *PACKAGE_VERSION = "2.0.0";
const char *GIT_REV = "fe04cff+";
const char *GIT_TAG = "NONE";
const char *GIT_BRANCH = "dev";
const char *VERSION = "2.0.0";
root@photon-ea4f47d2e8d0 [ ~/tdnf/build ]# ./bin/tdnf --version
tdnf: 2.0.0
root@photon-ea4f47d2e8d0 [ ~/tdnf/build ]#
```

> why would there be a version change after cmake

It doesn't happen on CI runs, yes :-). But when we make dev-release artifacts (such as compiled binaries for debugging, sometime in future) from a recently tagged HEAD, this will be triggered. We can be careful and run cmake again before such releases, but it's to come up with a method to deal with that scenario too. Just trying to be careful here.